misk-zookeeper/src/main/kotlin/misk/clustering/zookeeper/ZkLease.kt

```diff
@@ -155,6 +172,26 @@ internal class ZkLease(
     return false
   }
 
+  private fun notifyAfterAcquire() {
```

Comment:
Should listeners be notified asynchronously?

Reply:
No, but we do need to be careful about possible deadlocks from app code calling back into the lease.